livre/core/data/VolumeDataSource.h

```diff
@@ -79,6 +79,12 @@ class VolumeDataSource : boost::noncopyable
      */
     LIVRECORE_API ConstLODNodePtr getNode( const NodeId nodeId ) const;
 
+    /**
+     * Updates the volume. For instance the streaming data sources
+     * the range can change in time.
```

Comment:
For example, data sources may update their temporal range based on newly available data.